### PR TITLE
Update GSoD-interest.md for 2021

### PR DESCRIPTION
**Pull request by vchrombie**
This commit updates the GSoD Interest page for the year 2021.

### CODE REVIEW

File: GSoD-interest.md [hunk 1]
```diff
@@ -1,43 +1,13 @@
-# Google Season of Docs 2020 Interested Candidates
+# Google Season of Docs 2021 Interested Candidates
 
-Hi potential GSoD students,
-
-If you are interested in participating in Season of Docs as a technical writer, first familiarize yourself with the technical writer guide at https://goo.gl/dMVSQ8 and the technical writer responsibilities at https://goo.gl/BEqo2A. Check the [GSoD 2020 Technical Writer Participant Agreement](https://developers.google.com/season-of-docs/terms/tech-writer-terms) too.
```
Comment (GeorgLink) [inline GSoD-interest.md:5]: Maybe keep this?

File: GSoD-interest.md [hunk 2]
```diff
@@ -1,43 +1,13 @@
...
-
-1) You must select one (or more) idea out of the five ideas which we have proposed. You can find the ideas at the GSoD Idea page at: [GSoD-ideas.md](./GSoD-ideas.md).
-
-2) Once you have selected the idea, create a pull request on this file below to add yourself, your information, and a link to your proposal. **UPDATE:** This repository now requires [Developer Certificate of Origin](https://developercertificate.org/) (DCO) sign-off; see [CONTRIBUTING.md](https://github.com/chaoss/governance/blob/master/CONTRIBUTING.md#code-or-document-change-contributions-github-interface) for details on how to sign your commits.
-
-3) You are welcome to include in your previous technical writing work, contributions to other projects, skills, and other related information.
-
-4) Using and submitting other people's work as your own is not allowed. If you use other people's work, be sure to acknowledge their work in your submission.
-
-You must complete these things by July 9, 2020 at 18:00 UTC. Make sure to also submit the [Technical Writer Application Form for 2020 Season of Docs](https://bit.ly/gsod-tw-app), provided by Google.
```
Comment (GeorgLink) [inline GSoD-interest.md:24]: Maybe update the process we follow for applicants this year?

**Review by GeorgLink — CHANGES_REQUESTED** [review]
How about we keep or update the instructions to inform applicants what they need to do?

File: GSoD-interest.md (hunk 1 above)
Comment (vchrombie) [reply GSoD-interest.md:5]: Okay :+1:

File: GSoD-interest.md [hunk 3]
```diff
@@ -1,43 +1,21 @@
...
+If you are interested in participating in Season of Docs as a technical writer, first familiarize yourself with the [Technical Writer Guide | Season of Docs](https://developers.google.com/season-of-docs/docs/tech-writer-guide).
 
-**NOTE**: 
-1. CHAOSS is participating in GSoD 2020 under The Linux Foundation, [Join CHAOSS for GSoD 2020](https://chaoss.community/blog-post/2020/06/09/google-summer-of-docs).
-2. You can also find the ideas on The Linux Foundation wiki page, [GSoD 2020 CHAOSS wiki](https://wiki.linuxfoundation.org/gsoc/2020-gsod-chaoss).
-3. Know more about the working groups from here, [Participate, CHAOSS Community](https://chaoss.community/participate/).
+This document serves the purpose of Statement of Interest which has to be submitted by the technical writers. Please read about it, [Creating a statement of interest | Season of Docs](https://developers.google.com/season-of-docs/docs/tech-writer-statement).
```
Comment (GeorgLink) [inline GSoD-interest.md:7]:
```suggestion
Adding a link to your document below serves the purpose of Statement of Interest which has to be submitted by the technical writers. Please read about it, [Creating a statement of interest | Season of Docs](https://developers.google.com/season-of-docs/docs/tech-writer-statement).
```

**Review by GeorgLink — CHANGES_REQUESTED** [review]
I just have one small minor edit and then this looks good

**Pull request merged**